### PR TITLE
[All Hosts] (snippets) Adding some high-value snippets

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
For the OfficeRuntime.Auth interface, I created a new file officeruntime-snippets.yaml on the pattern of the other files. I assume that's all that's needed.

### CODE REVIEW

File: docs/code-snippets/officeruntime-snippets.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Office.Auth:interface:
```
Comment (AlexJerabek) [inline docs/code-snippets/officeruntime-snippets.yaml:1]: Throughout the snippet, shouldn't it be `OfficeRuntime.Auth`?

**Review by AlexJerabek — APPROVED** [review]
Approving. I'll check that the snippet mapping works on the backend.

One meta question about `OfficeRuntime.Auth`: should that interface be deprecated? @davidchesnut to weigh in too.

File: docs/code-snippets/officeruntime-snippets.yaml (hunk 1 above)
Comment (Rick-Kirkham) [inline docs/code-snippets/officeruntime-snippets.yaml:1]:
```suggestion
OfficeRuntime.Auth:interface:
```

**Comment by Rick-Kirkham** [comment]
> Approving. I'll check that the snippet mapping works on the backend.
> 
> One meta question about `OfficeRuntime.Auth`: should that interface be deprecated? @davidchesnut to weigh in too.

It's used in the SSO overview: [SSO in Office Add-ins / Add client side code](https://learn.microsoft.com/en-us/office/dev/add-ins/develop/sso-in-office-add-ins?tabs=jsonmanifest#add-client-side-code). Also, I dimly recall that when Office.auth was created there was still one scenario in which OfficeRuntime.auth had to be used. Is there a scenario in which the Office object is not available, but OfficeRuntime is?

**Comment by AlexJerabek** [comment]
@Rick-Kirkham, looks like this isn't building correctly. I'm investigating and hope to have a fix soon.

@ElizabethSamuel-MSFT, for awareness.

**Pull request merged**